src/components/home/balance-card.tsx

```diff
@@ -76,44 +82,53 @@ const BalanceCard = () => {
     );
   };
 
-  return (
+  /**
+   * Render the card content.
+   */
+  const cardContent = (
```

Comment:
I would be inclined to make this a function named renderCardContent. But doesn't make much difference. If you did refactor, the function could also contain the developerMode check and return accordingly which might be a bit cleaner